download/releaseNotes/releaseNotes7.0.adoc

```diff
@@ -195,3 +195,82 @@ This example also demonstrates real-time producing and consuming of tasks.
 * The documentation is now in AsciiDoc, which improves output quality and makes it easier to contribute.
 * `HardMediumSoftBigDecimalScore`: 3 score levels of BigDecimal scores. Contributed by Brad Hards.
 * The public API is now a solid foundation for the 7.x years.
+
+== Workbench
+
+=== Enhanced solver editor
+
+OptaPlanner Solver editor screen now supports adding all termination types, including composite termination.
+Phase configuration section allows you to tweak Construction Heuristic settings and select Local Search alogorithm
+to optimize your planning problem.
+
+image:7.0/solverEditorScreen.png[Solver editor screen]
+
+=== Guided rule editor integration
+
+Remember the pain of defining a constraint match in rules created by Guided rule editor? The process required creating
+a global variable `scoreHolder` and using `Free form line` input to insert Java code to modify the score.
+
+[source,java,options="nowrap"]
+----
+scoreHolder.addSoftConstraintMatch(kcontext, -$computer.cost);
+----
```

Comment:
I wouldn't bother explaining the pain of "before" or showing old ways (in the upgrade recipe that can make sense, but this document is the good news show :)). People read this document diagonally, and they might not grasp that what you're showing is "before".
Instead, I 'd focus on explaining that we now have support in the GRE for adding constraints, without mentioning the past.